File: Python/Exercicio17.py
# Algoritmo que classifica um nadador em uma categoria baseado na idade.

# Entrada
idade = int(input("Informe sua idade: "))

# Processamento
if idade >= 5 and idade <= 7:
    print("Categoria Infantil A. ")
elif idade >= 8 and idade <= 11:
    print("Categoria Infantil B. ")
elif idade >= 12 and idade <= 13:
    print("Categoria Juvenil A. ")
elif idade >= 14 and idade <= 17:
    print("Categoria Juvenil B. ")
elif idade >= 18:
    print("Categoria Adulto")

File: C/Exercicio17.c
//Algoritmo que lê a idade de um nadador e classifica em uma categoria.
#include <stdio.h>

int main() {
    //Variável
    int idade;

    //Entrada
    printf("Qual a sua idade? \n");
    scanf("%d", &idade);

    //Processamento
    if (idade >= 5 && idade <= 7) {
        printf("Infantil A. \n");
    } else if (idade >= 8 && idade <= 11) {
        printf("Infantil B. \n");
    } else if (idade >= 12 && idade <= 13) {
        printf("Juvenil A. \n");
    } else if (idade >= 14 && idade <= 17) {
        printf("Juvenil B. \n");
    } else if (idade >= 18) {
        printf("Adulto. \n");
    }
    return 0;
}